From 16c652a9b11601b9f6b55a419a0f0195ba93fbc5 Mon Sep 17 00:00:00 2001
From: Christian Ambach <ambi@samba.org>
Date: Fri, 30 Sep 2011 17:07:05 +0200
Subject: [PATCH 232/321] s3-docs: some corrections for wbinfo

Parameters for --group-info and --gid-info were not listed
properly in the SYNOPSIS and the OPTIONS section

Autobuild-User: Christian Ambach <ambi@samba.org>
Autobuild-Date: Fri Sep 30 18:44:34 CEST 2011 on sn-devel-104
---
 docs-xml/manpages-3/wbinfo.1.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/docs-xml/manpages-3/wbinfo.1.xml b/docs-xml/manpages-3/wbinfo.1.xml
index f816b80df07..0a4e96f4953 100644
--- a/docs-xml/manpages-3/wbinfo.1.xml
+++ b/docs-xml/manpages-3/wbinfo.1.xml
@@ -33,8 +33,8 @@
 		<arg choice="opt">--getdcname domain</arg>
 		<arg choice="opt">--get-auth-user</arg>
 		<arg choice="opt">-G gid</arg>
-		<arg choide="opt">--gid-info</arg>
-		<arg choide="opt">--group-info</arg>
+		<arg choide="opt">--gid-info gid</arg>
+		<arg choide="opt">--group-info group</arg>
 		<arg choice="opt">--help|-?</arg>
 		<arg choice="opt">-i user</arg>
 		<arg choice="opt">-I ip</arg>
@@ -171,8 +171,8 @@
 		</varlistentry>
 
 		<varlistentry>
-		<term>--group-info <replaceable>user</replaceable></term>
-		<listitem><para>Get group info for user.
+		<term>--group-info <replaceable>group</replaceable></term>
+		<listitem><para>Get group info from group name.
 		</para></listitem>
 		</varlistentry>
 
-- 
2.14.3

